extensions/utf8-only.md

```diff
@@ -46,3 +46,6 @@ Server: ERROR :Your IRC software MUST use UTF-8 encoding on this network
 Client: PRIVMSG #ircv3 :<non-utf-8 message>
 Server: WARN PRIVMSG INVALID_UTF8 :Your message was not correctly encoded as UTF-8 and had to be modified
 ```
+
+## Implementation considerations
+Implementations MUST ensure that if they truncate messages to meet a length limit, they do not do so in the middle of a UTF-8 codepoint.
```

Comment:
This should be 'must' rather than 'MUST' - our implementation consideration sections are non-normative and so the `MUST` and `SHOULD` and official-uppercasey-terms can't be used in 'em (don't you love some spec pedantry ;D ).